hms-app/src/components/card-details/IdeaDetails.tsx

```diff
@@ -389,15 +419,16 @@ export default function IdeaDetails(props: IProps) {
                   <Text className={classes.label}>Goal</Text>
                   <Text className={classes.text}>{ideaData.goal}</Text>
                 </Card.Section>
-
+                {/* 
+                                <Card.Section className={classes.borderSection}>
+                                    {skillsRequared}
+                                </Card.Section> */}
```

Comment:
could probably drop this comment, or at least leave a TODO or something describing why it is commented out